pkg/instance/ttlDestroyConsumer.go

```diff
@@ -43,23 +46,28 @@ func (c *ttlDestroyConsumer) Consume() error {
 			return
 		}
 
-		err := c.instanceDeleter.Delete(payload.ID)
+		instance, err := c.instanceService.FindDeploymentInstanceById(payload.ID)
```

Comment:
Why do we search it first? Can't we just delete it and log the error which in case it could not be found will say it did not exist 🤔